Src/Base/AMReX_FabArrayBase.cpp

```diff
@@ -1063,7 +1090,7 @@ FabArrayBase::getFB (const IntVect& nghost, const Periodicity& period,
     }
 
     // Have to build a new one
-    FB* new_fb = new FB(*this, nghost, cross, period, enforce_periodicity_only);
+    FB* new_fb = new FB(*this, nghost, cross, period, enforce_periodicity_only,multi_ghost);
```

Comment:
Before this, we look up in the FB cache.  That code needs modification too to include test on `multi_ghost`.

Reply:
And we need to save a copy of `multi_ghost` in `FB`.

Reply:
Done. I ended up actually storing `multi_ghost` in `FabArrayBase` as well as `FB`. Now `multi_ghost` will get set as a property of the `FabArrayBase` object and should be used consistently every time `getFB` is called. (Consequently I also removed the `multi_ghost` argument from all the `FillBoundary` calls.)

Eventually it would be nice to remove the `multi_ghost` entirely and just do this for all multifabs that have more than one ghost node. But I don't know what that would end up breaking.

Reply:
I don't think we want to do that by default for all MulitiFabs.  There is cost associated with doing this.